examples/material_gallery/lib/demo/flexible_space_demo.dart

```diff
@@ -113,11 +113,11 @@ class FlexibleSpaceDemoState extends State<FlexibleSpaceDemo> {
               itemBuilder: (BuildContext context) => <PopupMenuItem<AppBarBehavior>>[
                 new PopupMenuItem<AppBarBehavior>(
                   value: AppBarBehavior.scroll,
-                  child: new Text('AppBar scrolls away')
+                  child: new Text('Toolbar scrolls away')
```

Comment:
I have a feeling this doesn't matter, so don't change it. The "app bar" term, along with some diagrams that imply that app bar's contain toolbars, can be found here: https://www.google.com/design/spec/patterns/scrolling-techniques.html

It's not all perfectly consistent but there is this:
> The standard app bar is the overarching component that can include the following blocks: a toolbar, tab bar, or flexible space.